### ISSUE TITLE
I think I am having similar issues, please help.  I also run into issues when running app.py

### ISSUE THREAD

**Issue by njay98** [dongido001/python-realtime-table#2]
I think I am having similar issues, please help.  I also run into issues when running app.py 
It mentions that I have a NoneType error but the field is certainly a string. I also have everything intalled correctly. Please advise 

![cap](https://user-images.githubusercontent.com/52171845/61547364-da4a1e80-aa10-11e9-9637-e542878edd9b.PNG)
![Capture](https://user-images.githubusercontent.com/52171845/61547365-da4a1e80-aa10-11e9-81f0-1bc3cb8a6562.PNG)
![Capture2](https://user-images.githubusercontent.com/52171845/61547366-da4a1e80-aa10-11e9-8a49-379346906829.PNG)

_Originally posted by @njay98 in https://github.com/dongido001/python-realtime-table/issues/1#issuecomment-513275342_

**Comment by dongido001** [comment]
@njay98, do you have [Pusher](https://dashboard.pusher.com/accounts/sign_up) [keys](https://dashboard.pusher.com/accounts/sign_in) set in your .env file?

**Comment by njay98** [comment]
@dongido001 Yes I do. I pulled this from the app keys from the pusher website and all 4 fields are inserted.

**Comment by dongido001** [comment]
Okay, that's weird. Do you have some time for TeamViewer, so I can take a look?

**Comment by njay98** [comment]
Sorry, I was unavailable over the weekend but I will this week (today included). Although I wont be able to use TeamViewer, would discord work?

**Comment by abescott** [comment]
I too am receiving the "should be a string" message

**Comment by dongido001** [comment]
The error is probably from your `.env` file, make sure you have this set up properly and restart the app anytime you change the keys in the `.env` file
